chrome-app/links.js

```diff
@@ -18,7 +18,7 @@ const ourRepos = ["repo:aws/jsii", "repo:aws/aws-cdk"];
 // LINKS
 const LINKS = [
     [
-        { title: "Review", href: "https://github.com/pulls/review-requested" },
+        { title: "Review", href: gitHubQuery("https://github.com/pulls/review-requested", [  "is:open", "archived:false", "label:contribution/core" ]) },
```

Comment:
I'm entirely sure that I agree with this one. 

* What about situations where you are explicitly called in for a review on a community PR?
* If you've reviewed a PR once, you will be part of the reviewers, and the PR will continue to show up (seems logical that you want to continue having your say in the futher evolution).
* What about non-CDK repos (jsii, cdk-ops, whatever), where you may be assigned to review but we don't add the label `contribution/core`?

What specific problems is this causing for you? Personally, I don't find it a problem that a single PR may occur in multiple quick-lists. Either you've already looked at them in a different list and they are no longer marked "unread", or they aren't and they deserve your attention for multiple reasons.

Seems you want to practice some kind of "inbox zero" style work here, right? Maybe we should inject a script that hides "read" items from the list? (There is no `is:unread` query unfortunately)

Reply:
Case in point: I only saw this PR because it appeared in my "To Review" list ;)

Reply:
Yes, inbox zero is my goal. It drives me crazy to keep scanning these lists and having to determine if there's something I need to do there or not.